### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mrogozhinskaya**

### CODE REVIEW

**Comment by mrogozhinskaya** [comment]
>

**Comment by mrogozhinskaya** [comment]
- [DEMO LINK](https://mrogozhinskaya.github.io/layout_html-form/)
- [TEST REPORT LINK](mrogozhinskaya.github.io/layout_html-form/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,117 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1></h1>
```
Comment (OleziO) [inline src/index.html]: Remove the empty tag.

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,117 @@
...
+        <legend>Personal information:</legend> 
+        <label class="form-field">
+          Surname:
+          <input type="text" name="Surname:" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text"
```
Comment (OleziO) [inline src/index.html]: Move each property to a new line with two lines with 2 spaces before.

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,117 @@
...
+          <input type="text" 
+          name="Name:" required
+          >
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input type="number" name="How old are you?" 
+          value="12"
+          min="1"
+          max="100"
```
Comment (OleziO) [inline src/index.html]: Same as the previous comment.

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,117 @@
...
+        </label>
+        <label class="form-field">
+        Full date of birth:
+          <input type="date" name="Full date of birth:" 
+          required placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="form-field">
+          <input type="checkbox">
+        I accept the term of the agreement
```
Comment (OleziO) [inline src/index.html]: Add 4 spaces.

**Review by OleziO — CHANGES_REQUESTED** [review]
Fix radio buttons logic
image

File: src/index.html (hunk 4 above)
Comment (mrogozhinskaya) [reply src/index.html]: The task is ready. Please check.

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,115 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/" 
+    method="post" 
+    autocomplete="off">
```
Comment (AlisaVlasova) [inline src/index.html]:
```suggestion
    <form
      action="http://localhost:8080/" 
      method="post" 
      autocomplete="off"
    >
```

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,115 @@
...
+          <input type="text" name="Surname:" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text" name="Name" 
+          required
+          >
+        </label>
```
Comment (AlisaVlasova) [inline src/index.html]:
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="Surname:" 
            required
          >
        </label>
        <label class="form-field">
          Name:
          <input
            type="text"
            name="Name" 
            required
          >
        </label>
```
fix formatting everywhere

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,115 @@
...
+        Do you love cats?
+          <input type="radio" name="answer" 
+          value="yes"
+          >
+        Yes
+          <input type="radio" name="answer" 
+          value="no"
+          >
+        No
+        </label>
```
Comment (AlisaVlasova) [inline src/index.html]:
```suggestion
        <div class="form-field">
          Do you love cats?
          <label>
            <input
              type="radio"
              name="answer" 
              value="yes"
            >
            Yes
          </label>
       
          <label>
            <input
              type="radio"
              name="answer" 
              value="no"
            >
            No
          </label>
        </div>
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Comment by mrogozhinskaya** [comment]
Done!

**Review by mrogozhinskaya — COMMENTED** [review]
The task is completed.

**Pull request closed without merging**